packages/fx-core/src/core/middleware/projectMigratorV3.ts

```diff
@@ -122,6 +122,15 @@ export const ProjectMigratorMWV3: Middleware = async (ctx: CoreHookContext, next
       ctx.result = ok(undefined);
       return;
     }
+    if (!isMigrationV3Enabled()) {
+      TOOLS?.logProvider.warning(
+        `The current project is not supported by this Teams Toolkit version. Please install the old version Teams Toolkit ${
```

Comment:
Can we tell user to create new project using this version to try latest feature?